contrib/test_jenkins.sh

```diff
@@ -693,6 +695,12 @@ run_mpi_tests() {
 
 			test_malloc_hooks_mpi
 
+			if [ "X$have_cuda" == "Xyes" ] && [ -x ./test/mpi/test_mpi_cuda ]
+			then
+				echo "==== Running MPI CUDA tests ===="
+				${MPIRUN_COMMON} -np 2 ./test/mpi/test_mpi_cuda
```

Comment:
can we run it several times with different env vars? E.g.
- UCX_TLS=^cuda_ipc // to test cuda_copy
- UCX_RNDV_SHEME=put_ppln
maybe some others?

Reply:
Added test variants.